examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -102,6 +108,10 @@ app::Clusters::ModeSelect::StaticSupportedModesManager sStaticSupportedModesMana
 #ifdef CONFIG_CHIP_CRYPTO_PSA
 chip::Crypto::PSAOperationalKeystore sPSAOperationalKeystore{};
 #endif
+
+#ifdef CONFIG_NET_L2_OPENTHREAD
+app::Clusters::NetworkDriverObj<DeviceLayer::NetworkCommissioning::GenericThreadDriver> threadNetworkDriver(0 /*endpointId*/);
```

Comment:
```suggestion
Clusters::NetworkDriverObj<DeviceLayer::NetworkCommissioning::GenericThreadDriver> threadNetworkDriver(0 /*endpointId*/);
```
but see below about how this is probably the wrong namespace.